include/random/rand32.h

```diff
@@ -72,6 +73,31 @@ extern void sys_rand_get(void *dst, size_t len);
  */
 extern int sys_csrand_get(void *dst, size_t len);
 
+
+/**
+ * @brief Allow user thread access to random API.
+ *
+ * Random API that uses entropy device requires that the user thread has
+ * granted access to the entropy driver. This API is a helper that should be
+ * called before a user thread uses any other random API.
+ *
+ * @note This function is not necessary if the thread was already granted
+ * access to the entropy device.
```

Comment:
Maybe mention that this isn't necessary either if the underlying random implementation doesn't use an entropy driver.